tests.sh

```diff
@@ -238,6 +238,8 @@ build_objectivec_osx() {
 build_objectivec_cocoapods_integration() {
   # First, load the RVM environment in bash, needed to update ruby.
   source ~/.rvm/scripts/rvm
+  # Update rvm to the latest version.
```

Comment:
Can you add why we need to do this?  (so we maybe can update it in the future to a fixed version)